routes/login.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const User = require('../models/User');
+const router = express.Router();
+
+// create a response for the homepage
+router.post('/', (req, res , next) => {
+    User.create(req.body, (err ,user) => {
+        if(err){
+            res.json(
+              {
+                configuration: 'fail',
+                error : err,
+            }  
+            );
+            return;
+        }
+        // when there is no error
```

Comment:
The comment does not match the next line. The error is handled above. Suggestion: remove the comment, I think the code is explicit enough.